perun-openapi/openapi.yml

```diff
@@ -1480,6 +1480,14 @@ components:
       in: query
       required: true
 
+    extSourceId:
```

Comment:
I would maybe consider renaming this to `sourceId` since someone might expect the name being `extSource` instead of the `source`.